.circleci/config.yml

```diff
@@ -66,8 +66,8 @@ jobs:
       - run: make check-generated
       - run: make test TEST_FLAGS="-race -timeout 5m"
       - run: make all
-      - e2e_tests:
-          helm_version: v2
+      # - e2e_tests:
+      #     helm_version: v2
```

Comment:
@stefanprodan sorry for the churn, once https://github.com/stefanprodan/podinfo/pull/69 is released, I will reenable the Helm 2 tests and submit this PR for review.